src/parsers/power_system_table_data.jl

```diff
@@ -13,6 +13,7 @@ struct PowerSystemTableData
     gen::Union{DataFrames.DataFrame,Nothing}
     load::Union{DataFrames.DataFrame,Nothing}
     services::Union{DataFrames.DataFrame,Nothing}
+    storage::Union{DataFrames.DataFrame,Nothing}
```

Comment:
Is this inflow?

Reply:
Thats for the `storage.csv` file in data/RTS_GMLC which currently holds :
- Max Volume GWh,
- Initial Volume GWh,
- Start Energy  GWh,
I think Inflow would be in  the data/forecasts